packages/legend-studio/src/stores/LegendStudioPlugin.ts

```diff
@@ -68,10 +69,20 @@ export type ModelLoaderExtensionConfiguration = {
   renderer: (editorStore: EditorStore) => React.ReactNode | undefined;
 };
 
-export type TestableMetadataGetter = (
+export type TestableElementEditorOpener = (
   testable: Testable,
   editorStore: EditorStore,
-) => TestableMetadata | undefined;
+) => TestableElementEditorState | undefined;
+
+export type TestableExtension = {
```

Comment:
--> `TestableSupport`, also to not crowd up this file, we should move this to where we have `TestableMetadata` right now, i.e. `GlobalTestRunnerState`